manifests/m/Microsoft/Office/16.0.14729.20228/Microsoft.Office.installer.yaml

```
@@ -0,0 +1,24 @@
+# Automatically updated by the winget bot at 2022/Jan/14
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.Office
+PackageVersion: 16.0.14729.20228

```

I'm getting this version but since its lower i'm accepting it
```suggestion
PackageVersion: 16.0.14729.20260
```